lib/cli/commands/config.js

```diff
@@ -0,0 +1,76 @@
+import path from "node:path";
+import os from "node:os";
+import chalk from "chalk";
+import baseMiddleware from "../middlewares/base.js";
+import Configuration from "@ui5/project/config/Configuration";
+
+const ALLOWED_KEYS = ["mavenSnapshotEndpointUrl"];
+const ui5RcFilePath = path.resolve(path.join(os.homedir(), ".ui5rc"));
+
+const configCommand = {
+	command: "config",
+	describe: "Configures the `.ui5rc` file.",
```

Comment:
Should we also mention somewhere in the command help that the file is located at `~/.ui5rc`. Otherwise people might think it can or is within the cwd?

Reply:
I actually intended to remove the filename from all descriptions. I don't see a great benefit in knowing the location. Neither git nor npm mention the specific file in use.